branchLayout/api/src/main/java/com/virtuslab/branchlayout/api/BranchLayout.java

```diff
@@ -68,12 +69,11 @@ public IBranchLayout slideIn(String parentBranchName, IBranchLayoutEntry entryTo
     return new BranchLayout(newRootEntries.map(rootEntry -> slideIn(rootEntry, entryToSlideIn, parentEntry)));
   }
 
-  private static boolean isDescendant(IBranchLayoutEntry presumedAncestor, IBranchLayoutEntry presumedDescendant) {
+  private static boolean isDescendant(@NonNull IBranchLayoutEntry presumedAncestor, IBranchLayoutEntry presumedDescendant) {
```

Comment:
`@NonNull` is always redundant
```suggestion
  private static boolean isDescendant(IBranchLayoutEntry presumedAncestor, IBranchLayoutEntry presumedDescendant) {
```